index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Update your readme and explaining your project. Currently, you have just copied/pasted the Microverse template. Please change the content so it matches your project description.

Reply:
README.md was updated.